rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +18,22 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise exceptions.ValidationError(msg)
+                    raise exceptions.ValidationError(
+                        msg,
+                        code='authorization'
```

Comment:
(Because easiest to review impact pros/cons of each when considered in isolation)